Tests/UnitTests/Paywalls/Events/PaywallEventStoreTests.swift

```diff
@@ -33,7 +33,7 @@ class PaywallEventStoreTests: TestCase {
     // - MARK: -
 
     func testCreateDefaultDoesNotThrow() throws {
-        _ = try PaywallEventStore.createDefault()
+        _ = try PaywallEventStore.createDefault(documentsDirectory: nil)
```

Comment:
This verifies that creating it with the default documents directory works across iOS versions.